ci-operator/config/redhat-developer/rhdh/.claude/skills/rhdh-aks-lifecycle/scripts/check-aks-lifecycle.sh

```
@@ -0,0 +1,73 @@
+
+set -euo pipefail
+
+AKS_API_URL="https://releases.aks.azure.com/parsed_data.json"
+EOL_API_URL="https://endoflife.date/api/azure-kubernetes-service.json"
+MAPT_REF=""
+TEST_PATTERN=""
+CONFIG_DIR="ci-operator/config/redhat-developer/rhdh"
+
+while [[ $# -gt 0 ]]; do
+  case "$1" in
+    --mapt-ref)       MAPT_REF="$2"; shift 2 ;;
```

Shellcheck (0.11.0)

[warning] MAPT_REF appears unused. Verify use (or export if used externally).

(SC2034)